### DISCUSSION TITLE
Unknown supported locales

### DISCUSSION THREAD

**Discussion by KasparRosin** [sergiodxa/remix-i18next#91] [Q&A]
I have an issue where I do not know the locales before I make a request inside root.
Currently there is no way of defining locales before, as it will break the remix-i18next getLocale function.
I also can't define them inside root loader, since it will not always run according to remix.
remix-i18next also does not accept empty/false for supportedLocales

**Comment by sergiodxa** [comment]
I'm not sure I understand, but you want to dynamically define the locales your app support?
